ansible/roles/ocp4-workload-dil-agile-integration/tasks/workload.yml

```diff
@@ -0,0 +1,86 @@
+---
+# - name: Provision Shared Resources
+#   include_tasks: provision_shared.yaml
+#   vars:
+#     shared_namespace: "shared"
+
+# - name: Provision Gogs
+#   include_tasks: provision_gogs.yaml
+#   vars:
+#     gogs_project: "gogs"
+
+# # Microcks operator needs Strimzi to be installed in the namespace (even though Async is not used)
+# - name: Provision AMQ Streams
+#   include_tasks: provision_streams.yaml
+#   vars:
+#     operators_project: "workshop-operators"
+
+# - name: Provision AMQ Online
+#   include_tasks: provision_online.yaml
+#   vars:
+#     operators_project: "workshop-operators"
+
+
+# - name: Provision User Registration
+#   include_tasks: provision_user_registration.yaml
+#   vars:
+#     registration_namespace: "registration"
+
+# - name: Provision Eclipse Che
+#   include_tasks: provision_che.yaml
+#   vars:
+#     che_project: "che"
+
+# - name: Provision 3scale
+#   include_tasks: provision_threescale.yaml
+#   vars:
+#     threescale_project: "threescale"
+
+# - name: Provision SSO
+#   include_tasks: provision_sso.yaml
+#   vars:
+#     sso_project: "sso"
+
+# - name: Provision Apicurito
+#   include_tasks: provision_apicurito.yaml
+#   vars:
+#     apicurito_project: "apicurito"
+
+# - name: Provision Microcks
+#   include_tasks: provision_microcks.yaml
+#   vars:
+#     microcks_project: "microcks"
+
+# - name: Provision Camel K
+#   include_tasks: provision_camelk.yaml
+#   vars:
+#     operators_project: "workshop-operators"
+
+
+# - name: Provision Fuse Online
+#   include_tasks: provision_fuse.yaml
+#   vars:
+#     fuse_project: "fuse"
+
+# - name: Provision Backend Project
+#   include_tasks: provision_backend.yaml
+#   vars:
+#     backend_project: "international"
+
+
+# - name: Provision User Namespaces
+#   include_tasks: provision_user_projects.yaml
+#   with_sequence: start=1 end={{ num_users }} format=%d
+#   vars:
+#     shared_namespace: "shared"
+#     sso_project: "sso"
+#     fuse_project: "fuse"    
+
+- name: Provision Solution Explorer
+  include_tasks: provision_webapp.yaml
+  vars:
+    webapp_namespace: "webapp"
+    configmap_name: "extra-services"
+    apicurito_project: "apicurito"
+    microcks_project: "microcks"
+    fuse_project: "fuse"    
```

Comment:
Looks like extra whitespace at the end of this line? Please look for and fix trailing whitespace errors.

Reply:
Still some trailing space here.